src/wasm_offset_converter.js

```diff
@@ -0,0 +1,73 @@
+function WasmOffsetConverter(wasmBytes) {
```

Comment:
a comment explaining what this does would be useful I think. i started to read the body but i realized it's hard for me to guess at what the goal is here.

Reply:
I'll add some documentation.